openquake/hazardlib/geo/surface/simple_fault.py

```diff
@@ -171,17 +172,22 @@ def from_fault_data(cls, fault_trace, upper_seismogenic_depth,
 
     @classmethod
     def get_fault_vertices_3d(cls, fault_trace, upper_seismogenic_depth,
-                              lower_seismogenic_depth, dip):
+                              lower_seismogenic_depth, dip, index_patch=None):
         """
-        Get surface main vertexes.
-
+        Get surface main vertices.
         Parameters are the same as for :meth:`from_fault_data`, excluding
         mesh spacing.
 
+        :param index_patch:
+            indicate the patch of the fault to output the vertices.
+            The first patch strating from where the fault trace starts.
         :returns:
```

Comment:
If index_patch is None, it returns the coordinates of the points delimiting the border of the fault surface as three :class:`numpy.array` containing longitudes, latitudes, depths, respectively. The order of vertexes is clockwise.
If index_patch is not None, returns four :class:`~openquake.hazardlib.geo.point.Point` objects representing the four vertices of the target patch.